nest/03.nest.md

```diff
@@ -0,0 +1,88 @@
+
+`nest` will print an error message if more than one package matches the syntax you wrote.
+In that case, you can go on [the official Raven-OS website](https://stable.raven-os.org) and change the syntax accordingly.
+
+For example, the output of `nest install stable::sys-bin/ncurses6-bin` is as followed:
+```
+$ nest install ncurses6-bin
+4 pending transactions:
+
+ install stable::sys-lib/libncurses6-term#6.1.0
+ install stable::sys-lib/libc6#2.28.0
+ install stable::sys-lib/libncursestw6#6.1.0
+ install stable::sys-bin/ncurses6-bin#6.1.0
+
+Would you like to apply these transactions? [Yes/no]
+```
+
+As you can see, the last package listed above is the package you initially wanted to install, and above it are all the dependencies that it has and that are not already installed on the OS.
+You will then be prompted with the question `Would you like to apply these transactions?`, which you can answer with either '*yes*', '*Yes*', '*Y*', '*y*', or '*no*', '*No*', '*n*', '*N*'.
+
+An important thing to note, is that when Nest (and most CLI programs) ask you a closed question like this one, the default answer usually begins with an uppercase letter.
+So in this case, just pressing enter is equivalent to typing `yes` and then pressing enter.
+
+#### Uninstalling a package
+The command to uninstall a package is `nest uninstall <package>`.
+
+It will remove the given package, its binaries, libraries, manual pages...
+Basically everything that was installed with it, but it will keep all the files generated by the package itself.
+So, if this package was a text editor for example, the files you created or edited with it won't be altered if you uninstall the package.
+
+However, this won't remove the package's dependencies, because they might still be used by another installed package.
+
+#### Common issues
+* `error: no package found for requirement <package>`
+
+This error could mean that you want to install a package that isn't in Nest's cache, so, `nest` can't find it and install it.
+Try running `nest pull` and then running your command again.
+
+Another possible explanation is that the package you wanted to install can't be found.
+You should check the package name syntax, in case you mistyped the repository, the category, the package name itself or the version.
+You can always check [on Raven-OS website](https://stable.raven-os.org/) as mentioned above, to have not make any mistakes.
+
+* `error: unable to find an installed package matching <package>`
+
+The package you wanted to uninstall can't be found.
+You have to check the package name syntax for any typo, and that you already have this package installed.
+
+### 2.3 Summary
+This concludes the part of this getting-started dedicated to `nest`!
+To be followed are some advanced concepts that are useful, but not necessary, to use `nest`.
+The readers interested in using `finest` should read this next chapter as well, because those concepts are important in order to use `finest`.
```

Comment:
```suggestion
Readers interested in using `finest` should read the next chapter as well, because those concepts are important in order to use `finest`.
```